datalab/mlalpha/_cloud_predictor.py~48ec5ba7df5b895901affdea7c5ca5d14e688a4f

```diff
@@ -0,0 +1,94 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
This file looks like a backup file created by some editor, and shouldn't be included.